runtime/src/bucket_map_holder_stats.rs

```diff
@@ -14,6 +14,10 @@ pub struct BucketMapHolderStats {
     pub entries_from_mem: AtomicU64,
     pub entry_missing_us: AtomicU64,
     pub entries_missing: AtomicU64,
+    pub load_disk: AtomicU64,
+    pub load_disk_us: AtomicU64,
+    pub load_disk_missing: AtomicU64,
+    pub load_disk_missing_us: AtomicU64,
```

Comment:
It look me a bit to understand the differences here. What about slightly different names?

- `load_disk_found_us` for the measurement when the item is found
- `load_disk_found_count` for the count of items found
- `load_disk_missing_us` for the measurement when the item is not found
- `load_disk_missing_count` for the count of items not found

Reply:
done